app/src/NativeBase/Components/Forms/TextInputControl.tsx

```diff
@@ -0,0 +1,157 @@
+import MaterialIcons from 'react-native-vector-icons/MaterialIcons'
+
+interface TextInputControlProps {
+  error: boolean
+  required: boolean
+  errorType?: 'invalid' | 'missing'
+  label: string
+  onBlur?: () => void
+  onChange: (value: string) => void
+  type: 'email' | 'firstName' | 'lastName'
+  value: string
+}
+
+const TextInputControl: FC<TextInputControlProps> = ({
+  error,
+  errorType,
+  label,
+  onBlur,
+  onChange,
+  required,
+  type,
+  value,
+}) => {
+  let autoCapitalize = 'none' as
+    | 'none'
+    | 'sentences'
+    | 'words'
+    | 'characters'
+    | undefined
+  let autoComplete = 'off' as
+    | 'birthdate-day'
+    | 'birthdate-full'
+    | 'birthdate-month'
+    | 'birthdate-year'
+    | 'cc-csc'
+    | 'cc-exp'
+    | 'cc-exp-day'
+    | 'cc-exp-month'
+    | 'cc-exp-year'
+    | 'cc-number'
+    | 'email'
+    | 'gender'
+    | 'name'
+    | 'name-family'
+    | 'name-given'
+    | 'name-middle'
+    | 'name-middle-initial'
+    | 'name-prefix'
+    | 'name-suffix'
+    | 'password'
+    | 'password-new'
+    | 'postal-address'
+    | 'postal-address-country'
+    | 'postal-address-extended'
+    | 'postal-address-extended-postal-code'
+    | 'postal-address-locality'
+    | 'postal-address-region'
+    | 'postal-code'
+    | 'street-address'
+    | 'sms-otp'
+    | 'tel'
+    | 'tel-country-code'
+    | 'tel-national'
+    | 'tel-device'
+    | 'username'
+    | 'username-new'
+    | 'off'
+    | undefined
+  let textContentType = 'none' as
+    | 'none'
+    | 'URL'
+    | 'addressCity'
+    | 'addressCityAndState'
+    | 'addressState'
+    | 'countryName'
+    | 'creditCardNumber'
+    | 'emailAddress'
+    | 'familyName'
+    | 'fullStreetAddress'
+    | 'givenName'
+    | 'jobTitle'
+    | 'location'
+    | 'middleName'
+    | 'name'
+    | 'namePrefix'
+    | 'nameSuffix'
+    | 'nickname'
+    | 'organizationName'
+    | 'postalCode'
+    | 'streetAddressLine1'
+    | 'streetAddressLine2'
+    | 'sublocality'
+    | 'telephoneNumber'
+    | 'username'
+    | 'password'
+    | 'newPassword'
+    | 'oneTimeCode'
+    | undefined
+
+  switch (type) {
+    case 'email':
+      autoComplete = 'email'
+      textContentType = 'emailAddress'
+      break
+    case 'firstName':
+      autoCapitalize = 'words'
+      autoComplete = 'name-given'
+      textContentType = 'givenName'
+      break
+    case 'lastName':
+      autoCapitalize = 'words'
+      autoComplete = 'name-family'
+      textContentType = 'familyName'
+      break
+  }
+
+  let text = ''
+
+  switch (errorType) {
+    case 'invalid':
+      text = "Please check - this doesn't look right"
+      break
+    case 'missing':
+    default:
+      text = 'Please complete this'
+      break
+  }
+
+  return (
+    <FormControl marginBottom="6" isRequired={required} isInvalid={error}>
+      <FormControl.Label>{label.toString()}</FormControl.Label>
+      <Input
+        accessibilityLabel={`${label} text`}
```

Comment:
Thanks for switching this to use `FormControl`.  Now you have, from what I saw in the docs I think using `FormControl.Label` is enough for accessibility, don't need `accessibilityLabel` as well (unless you've seen different guidance?)